TEST_APPS/device/nfcapp/SmartPoster.cpp

```diff
@@ -0,0 +1,247 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018-2018 ARM Limited
+ *
```

Comment:
can you add SPDX identifier (examples can be found in platform files for instance) to all new files

Reply:
updated the copyright - note: smartposter.cpp and smartposter.h will require moving into the actual O/S later